Sources/String+Sweetness.swift

```diff
@@ -37,6 +37,10 @@ public extension NSRange {
 }
 
 public extension String {
+    public var wholeRange: Range<String.Index> {
+        return self.range(of: self)!
```

Comment:
what if the string is empty ?

Reply:
should we force unwrap ? ..

Reply:
Definitely worth adding a test case on empty string

Reply:
Added test. Was surprised to learn that it does crash.

👍